changelogs/unreleased/gh-9895-forbid-empty-replicaset.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/config
+
+* Added additional validation to cluster configuration.
+  Now it is forbidden to create an empty replicaset (gh-9895).
```

Comment:
```suggestion
* Added additional validation to a cluster's configuration.
  Now it is forbidden to create an empty replica set (gh-9895).
```

Reply:
Applied the first line diff. I'd like to leave `replicaset` as a single word, similar to config schema.

Reply:
I use 'replicaset' as a single word too in comments in the code. I know that our documentation on the website uses 'replica set'. I remember that I changed my wording in changelogs to 'replica set' on a request several times.

However, I see both ways in existing release notes on GitHub: there is no consistency.

Can't suggest anything certain. I propose to don't spin here and go ahead.

@andreyaksenov I'm closing this discussion thread. I'm OK to discuss some common vocabulary our of scope of this PR.